scripts/test/no_auto_migrations.sh

```diff
@@ -10,7 +10,7 @@ fail() {
 }
 
 # search for auto migrations excluded the preexisting one
-find */migrations/*_auto_*.py | grep -v "20170113_2133" > "$TMPFILE"
+find */migrations/*_auto_*.py > "$TMPFILE"
```

Comment:
Take or leave: while we're here, is there any reason we can't just store the output of `find` in a variable instead of doing this dance with a temp file that we always have to remove?